universal_transfer_operator/src/universal_transfer_operator/data_providers/__init__.py

```diff
@@ -0,0 +1,32 @@
+import importlib
+
+from airflow.hooks.base import BaseHook
+
+from universal_transfer_operator.constants import TransferMode
+from universal_transfer_operator.data_providers.base import DataProviders
+from universal_transfer_operator.datasets.base import Dataset
+from universal_transfer_operator.utils import TransferParameters, get_class_name
+
+DATASET_CONN_ID_TO_DATAPROVIDER_MAPPING = {
+    "s3": "universal_transfer_operator.data_providers.filesystem.aws.s3",
+    "aws": "universal_transfer_operator.data_providers.filesystem.aws.s3",
```

Comment:
To remove duplication you could do the following -- this is not strictly needed but just an fyi

```suggestion
    **{dict.fromkeys(["aws", "s3"], "universal_transfer_operator.data_providers.filesystem.aws.s3"},
```

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1492/commits/c2e08eaec89c463d8e83e7a8fa523231e56ede4f)